encoding/abi/abi.go

```diff
@@ -0,0 +1,223 @@
+package abi
+
+import (
+	"math/big"
+
+	"github.com/algorand/go-algorand/data/abi"
+)
+
+type Value struct {
```

Comment:
@algorandskiy mentioned using struct embedding (e.g. `type Value struct { abi.Value }`) to avoid having to declare methods again, such as `Encode()`.

But would a simple type assignment work as well? E.g. `type Value = abi.Value`

Reply:
Wow I looks like it works @ahangsu 
I tried type aliasing (`type Value abi.Value`) and it did not work

Reply:
That works, it can also remove the redundant method wrapper, leaving only necessary constructor, `Decode` and `TypeFromString`. Nice one :-)